Makefile

```diff
@@ -309,6 +309,7 @@ docker-images-jaeger-backend docker-images-jaeger-backend-debug: create-baseimg
 			--build-arg base_image=$(BASE_IMAGE) \
 			--build-arg debug_image=$(DEBUG_IMAGE) \
 			--build-arg TARGETARCH=$(GOARCH) \
+			--load \
```

Comment:
according to the [docs][], this is a shorthand for `--output=type=docker`. I would use the latter, it makes the command self-explanatory.

[docs]: https://docs.docker.com/engine/reference/commandline/buildx_build/

Reply:
also, I don't see `--load` it my `docker help build`, but I do see `--output` flag, so it's probably better for compatibility too.

Reply:
Actually both `--load` and `--output=type=docker` fail for my version of Docker. When you say "it complains", does it fail or just print a warning? The docs say that by default the build results are "exported to `docker images`", which sounds like what we want because we then publish them explicitly.

Reply:
I will make the changes requested here.
I have tried with both `docker version` client and server 20.10.10 and .11.

The warning I saw was 

```
WARN[0000] No output specified for docker-container driver. Build result will only remain in the build cache. To push result image into registry use --push or to load image into docker use --load 
```

This warning did not trouble me.  However, after a successful build, `docker images` didn't show any new images!  On the advice of the warning I added `--load` and repeated the _make_, after which I saw four newly built images.

I will dig into this tomorrow, perhaps I have misconfigured something.